app/middlewares/service_unavailable_middleware.rb

```diff
@@ -0,0 +1,16 @@
+class ServiceUnavailableMiddleware
+  def initialize(app)
+    @app = app
+  end
+
+  def call(env)
+    if FeatureFlag.active?(:service_unavailable_page)
+      lookup_context = ActionView::LookupContext.new(ActionController::Base.view_paths)
+      view = ActionView::Base.with_empty_template_cache.new(lookup_context, {}, ActionController::Base.new)
+      body = view.render(template: 'errors/service_unavailable', layout: 'layouts/error')
+      [503, {'Content-Type' => 'text/html'}, [body]]
```

Comment:
is it possible to return 200 statuscode for `/check` and `/integrations/` endpoints?
if not just 200 instead of 503?

Reply:
Good point, will try and exclude those endpoints from the 503